tarjdk.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
 set -e
 
-if [ "$BUILD_IOS" != "1" ]; then
+if [[ "$BUILD_IOS" != "1" ]]; then
 
 unset AR AS CC CXX LD OBJCOPY RANLIB STRIP CPPFLAGS LDFLAGS
-git clone https://github.com/termux/termux-elf-cleaner
+git clone --depth 1 -b 'v2.2.0' https://github.com/termux/termux-elf-cleaner
```

Comment:
Why not use the 2.2.**1** release ?

Reply:
> Why not use the 2.2.**1** release ?

that release introduced [a commit](https://github.com/termux/termux-elf-cleaner/commit/9578f2c4bcb3034acecab452f7180e63b51491da) i tried to avoid, as per [the commit message here](https://github.com/PojavLauncherTeam/android-openjdk-build-multiarch/pull/19/commits/ebfd5b7a6a5483070e7a432ac5caf9c1eb4ba61a)

going further causes [this](https://github.com/fukiame/android-openjdk-build-aarch64/actions/runs/7525768725/job/20482664536#step:4:4356) and the build fails

Reply:
[termux-elf-cleaner 2.2.1 release notes](https://github.com/termux/termux-elf-cleaner/releases/tag/v2.2.1)